app/services/usps_in_person_proofing/mock/proofer.rb

```diff
@@ -29,7 +29,7 @@ def request_enroll(applicant)
         Response::RequestEnrollResponse.new(res)
       end
 
-      def request_facilities(_location)
+      def request_facilities(_location, _is_enhanced_ipp)
```

Comment:
When testing with Sinatra I got the IPP mock list and not the EIPP mock list.  Updating the following, I successfully got the EIPP list after coming in from Sinatra (picking Enhanced IPP).


```
      def request_facilities(_location, _is_enhanced_ipp)
        if _is_enhanced_ipp
          parse_facilities(JSON.parse(Fixtures.request_eipp_facilities_response))
        else
          parse_facilities(JSON.parse(Fixtures.request_facilities_response))
        end
      end
      ```